Tasks/PackerBuildV0/DefaultTemplates/default.linux.template.json

```diff
@@ -59,7 +59,7 @@
             "cd /deployTemp",
             "ls",
             "chmod +x /deployTemp/{{user `package_name`}}/{{user `script_relative_path`}}",
-            "/deployTemp/{{user `package_name`}}/{{user `script_relative_path`}} {{user `script_arguments`}}"
+            ". /deployTemp/{{user `package_name`}}/{{user `script_relative_path`}} {{user `script_arguments`}}"
```

Comment:
is this required?

Reply:
Without the source command (.) at the start, the script file is not found . I think this could perhaps be due to the bug in version 1.2.4, mentioned in https://github.com/hashicorp/packer/issues/6358 . version 1.2.5 is now available and I think this may have been fixed in packer version 1.2.5. Will try with packer version 1.2.5 without the source command at the start and update status here.

Reply:
Same issue with 1.2.5 without the source at the start.
``` azure-arm: /tmp/script_4060.sh: 5: /tmp/script_4060.sh: /deployTemp/scripts/script1.sh: not found``` .
@bishal-pdMSFT Let me know if it is fine to have the source command at the start.

Reply:
this looks okay to me